src/app/core/shell/sidenav/sidenav.component.scss

```diff
@@ -181,3 +175,8 @@
     }
   }
 }
+.sidebar-full{
```

Comment:
```suggestion

.sidebar-full {
```

Reply:
@abhaychawla ok ill make the necessary changes.